packages/remix-node/package.json

```diff
@@ -3,6 +3,8 @@
   "description": "Node.js platform abstractions for Remix",
   "version": "1.0.6",
   "license": "MIT",
+  "main": "./index.js",
```

Comment:
Revert changes to node module as it's not even runable in ESM because of the React package.